src/main/java/strategy/handlers/Balance.java

```diff
@@ -0,0 +1,22 @@
+package strategy.handlers;
+
+import dao.StorageDaoImpl;
+import model.FruitModel;
+import strategy.Strategy;
+import strategy.validator.CommodityValidator;
+
+public class Balance implements Strategy, CommodityValidator {
```

Comment:
It is better to rename this class to smth like BalanceOperationHandler and implement intarface wiht a name OperationHandler, not Strategy. The name Strategy would better looks as an abstraction for StratregyImpl